Makefile

```diff
@@ -188,6 +195,9 @@ buildah-push: buildah-build
 install-federation:
 	./hack/install-federation.sh
 
+$(GOPATH)/bin/mockgen:
+	go get -u github.com/golang/mock/mockgen/...
```

Comment:
It looks like the vendored code wasn't commited.

What does go get do now in this case? I assume it knows to use the vendored code?

Reply:
The vendor directory was already present as we had the change in Gopkg.lock file somehow. Refer https://github.com/openshift/hive/tree/master/vendor/github.com/golang/mock